librz/include/rz_il/rz_il_reg.h

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: 2021 Florian Märkl <info@florianmaerkl.de>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZIL_REG_H
+#define RZIL_REG_H
+
+#include <rz_reg.h>
+
+typedef struct rz_il_reg_binding_t {
+	size_t regs_count;
+	char **regs; //< regs_count names of registers that are bound to variables of the same name
+} RzILRegBinding;
```

Comment:
maybe use RzPVector ?

Reply:
It should be immutable, so no reason to use RzPVector after the initial creation anymore imo.

Reply:
i would still use RzPVector, since the implementation of these bindings is a representation of RzVector

Reply:
It's different now anyway. They are not pointers anymore, but structs and the array gives us much more type-safety than RzVector (which is still used in building, but only there).